docs/requirements.txt

```
@@ -1,3 +1,3 @@
-sphinx==4.0.2
+sphinx
```

Is there a reason to not use the version?

Can upgrade to 4.5.0 or even 5.0.1 if it works.

```suggestion
sphinx==4.0.2
```

---

4.0.2 used a module called `types`, in which I assume was replaced with the `typing` module, so upgrading it to the latest(5.0.1) version worked, but I will try 4.5.0.

---

If 5.0.1 works, you can use `==5.0.1`